docs/website/docs/dlt-ecosystem/verified-sources/personio.md

```diff
@@ -0,0 +1,189 @@
+   the following command:
+   ```bash
+   dlt pipeline <pipeline_name> show
+   ```
+   For example, the `pipeline_name` for the above pipeline example is `personio`, you may also use
+   any custom name instead.
+
+For more information, read the [Walkthrough: Run a pipeline.](../../walkthroughs/run-a-pipeline)
+
+## Sources and resources
+
+`dlt` works on the principle of [sources](../../general-usage/source) and
+[resources](../../general-usage/resource).
+
+### Source `personio_source`
+
+This function initializes class `PersonioAPI` in "personio/helpers.py" and returns data resources
+like "employees", "absences", and "attendances".
+
+```python
+@dlt.source(name="personio")
+def personio_source(
+    client_id: str = dlt.secrets.value,
+    client_secret: str = dlt.secrets.value,
+    items_per_page: int = DEFAULT_ITEMS_PER_PAGE,
+) -> Iterable[DltResource]:
+```
+
+`client_id`: Generated ID for API access.
+
+`client_secret`: Generated secret for API access.
+
+`items_per_page`: Maximum number of items per page, defaults to 200.
+
+### Resource `employees`
+
+This resource retrieves data on all the employees in a company.
+
+```python
+ @dlt.resource(primary_key="id", write_disposition="merge")
+    def employees(
+        updated_at: dlt.sources.incremental[
+            pendulum.DateTime
+        ] = dlt.sources.incremental(
+            "last_modified_at", initial_value=None, allow_external_schedulers=True
+        ),
+        items_per_page: int = items_per_page,
+    ) -> Iterable[TDataItem]:
+```
+
+`updated_at`: The saved state of the last 'last_modified_at' value. It is used for
+[incremental loading](../../general-usage/incremental-loading).
+
+`items_per_page`: Maximum number of items per page, defaults to 200.
+
+Like the `employees` resource discussed above, other resources `absences` and `attendances` load
+data from the Personio API to your preferred destination.
+
+## Customization
+
+### Create your own pipeline
+
+If you wish to create your own pipelines, you can leverage source and resource methods from this
+verified source.
+
+1. Configure the pipeline by specifying the pipeline name, destination, and dataset as follows:
+
+   ```python
+   pipeline = dlt.pipeline(
+      pipeline_name="personio",  # Use a custom name if desired
+      destination="duckdb",  # Choose the appropriate destination (e.g., duckdb, redshift, post)
+      dataset_name="personio_data"  # Use a custom name if desired
+   )
+   ```
+
+   :::note 
+   To read more about pipeline configuration, please refer to our [documentation](../../general-usage/pipeline). 
+   :::
+
+1. To load employee data:
+
+   ```python
+   load_data = personio_source().with_resources("employees")
+   print(pipeline.run(load_data))
+   ```
+
+1. To load data from all supported endpoints:
+
+   ```python
+   load_data = personio_source().with_resources("employees", "absences", "attendances")
+   print(pipeline.run(load_data))
```

Comment:
can you please add one more example:
how to use start_date and end_date for attendances resource https://github.com/dlt-hub/verified-sources/blob/master/sources/personio/__init__.py#L100-L111

Reply:
as discussed, let's skip for now. Will update later.